LCLSGeneral/LCLSGeneral/POUs/Data/FB_LREALFromEPICS.TcPOU

```diff
@@ -82,8 +87,8 @@ tonValueTimeout();
 tonSeverityTimeout();
 bValid := (NOT tonValueTimeout.Q) AND 
 		  (NOT tonSeverityTimeout.Q) AND 
-		  (iLastValidSeverity = 0);
+          (iLastValidSeverity <= iMaximumValidSeverity);
 fValue := fLastValidValue;]]></ST>
     </Implementation>
   </POU>
-</TcPlcObject>
\ No newline at end of file
+</TcPlcObject>
```

Comment:
wat is this? Does this happen on every change?

Reply:
Oh a new-newline...?

Reply:
It's vim automatically inserting a newline at EOF 🤷‍♂️